Framework/include/QualityControl/MonitorObject.h

```diff
@@ -134,7 +133,7 @@ class MonitorObject : public TObject
   // TODO : maybe we should always be the owner ?
   bool mIsOwner;
 
-  ClassDefOverride(MonitorObject, 2);
+  ClassDefOverride(MonitorObject, 3);
```

Comment:
What is this for?

Reply:
the version of the class must be incremented if its definition changes. It enables ROOT to read types that evolve over time.